README.md

```diff
@@ -7,7 +7,7 @@ RFCs can be in raw, draft, stable, deprecated, or retired status.
 | Shortname                                                  | Name                                        | Status |
 |------------------------------------------------------------|---------------------------------------------|--------|
 | [1/KP](0001-kaa-protocol/README.md)                        | Kaa Protocol                                | Draft  |
-| [2/DCX](0002-data-collection-extension/README.md)          | Data Collection Extension protocol          | Draft  |
+| [2/DCP](0002-data-collection-extension/README.md)          | Data Collection Protocol                    | Draft  |
```

Comment:
Rename directory? extension -> protocol

Or, we can drop RFC names from the directory, and leave 0002 in there.